Clinical-Genomics-Lund/bonsai
# 99 remove n noel cgmlst alleles from frontend

This PR removes the n novel cgmlst alleles from the fronend page and only display the first digit for the n reads assigned by Kraken.

Close #99 

### How to setup and perform the tests

1. Setup bonsai and load a sample to the database.
2. Goto the sample page
3. Verify that the unit in the number of reads in the Species prediction card are designated with a single letter
4. Verify that the number of novel allele row have been removed from the cgmlst QC card.
